com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -112,6 +112,8 @@ internal static HDRenderPipeline currentPipeline
         ShaderVariablesXR m_ShaderVariablesXRCB = new ShaderVariablesXR();
         ShaderVariablesRaytracing m_ShaderVariablesRayTracingCB = new ShaderVariablesRaytracing();
 
+        internal ref ShaderVariablesGlobal GetShaderVariablesGlobalCB() => ref m_ShaderVariablesGlobalCB;
```

Comment:
We should probably not return a ref here. It's usually fine to make your own copy when needed. This will avoid weird unintended side effects.

Reply:
👍 I pushed the modifications